src/components/DegreeChipStrip.module.css

```
@@ -206,36 +206,36 @@
 
-.app-container[data-layout-tier="mobile"] .degree-chip-interval {
+:global(.app-container[data-layout-tier="mobile"]) .degree-chip-interval {
   font-size: 0.82rem;
 }
 
 /* Phase 2: Desktop compact chip strip — smaller chips, tighter padding.
    Targets ~82px strip height (down from ~104px), keeping chip glyph legible. */
-.app-container[data-layout-tier="desktop"] .degree-chip-strip {
+:global(.app-container[data-layout-tier="desktop"]) .degree-chip-strip {
   --chip-size: clamp(1.82rem, 2.45vw, 2.02rem);
   padding: 0.34rem 0.78rem 0.38rem;
```

Stylelint (17.7.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)